core/src/main/java/org/elasticsearch/action/admin/cluster/node/info/NodeInfo.java

```diff
@@ -240,6 +252,11 @@ public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
         out.writeVInt(version.id);
         Build.writeBuild(build, out);
+        if (totalIndexingBuffer == null) {
```

Comment:
why do we follow a different pattern here? should we write a boolean first and restore the `null` value like other fields?